stglib/tb.py

```diff
@@ -0,0 +1,249 @@
+from datetime import timedelta
+
+import pandas as pd
+import xarray as xr
+
+from .core import filter, qaqc, utils
+
+
+def read_header(filnam):
+    """Read header info from a TruBlue 255 .csv file to add to metadata."""
+
+    header = {}
+
+    f = open(filnam)
```

Comment:
I know we don't do it on the other instruments, but we should be opening files using `with open()...` instead of this way.

Reply:
Changed